src/backend/app/auth/auth_routes.py

```diff
@@ -83,19 +84,27 @@ async def callback(request: Request, osm_auth=Depends(init_osm_auth)):
     log.debug(f"Access token returned of length {len(access_token)}")
     response = Response(status_code=HTTPStatus.OK)
 
+    osm_user = osm_auth.deserialize_access_token(access_token)
+    user_data = {
+        "sub": f"fmtm|{osm_user['id']}",
+        "aud": "fmtm.localhost:8000",
```

Comment:
The actual audience via the proxy is fmtm.localhost:7050

You can build this using the FMTM_DOMAIN var and DEV_PORT vars, see a similar approach for building the URL here: https://github.com/hotosm/fmtm/blob/b143186a9368b41c8fa7a2c122e6dea003e5b0db/src/backend/app/config.py#L173

Hopefully if the audience matches from serialisation to deseralisation, this should work fine 👍

Reply:
ah crap I used that to test audience , but pushed it mistakenly. Thanks for pointing it out.

Reply:
I found that audience was failing because I failed to mention it while decoding the encoded token.
`audience = settings.FMTM_DOMAIN` like this while decoding.